x-pack/plugins/security_solution/common/endpoint/models/policy_config.ts

```diff
@@ -115,6 +117,24 @@ export const policyFactoryWithoutPaidFeatures = (
   };
 };
 
+/**
+ * Strips paid features from an existing or new `PolicyConfig` for gold and below license
+ */
+export const policyFactoryWithSupportedFeatures = (
```

Comment:
is there a reason why you created another function as opposed to just updating the existing policyFactory (policy config with defaults for platinum licenses) and the policyFactoryWithoutPaidFeatures (policy config with defaults for gold and below licenses) windows.ransomware.supported fields?

Reply:
I also use this function to correctly set the `supported` when the license changes.  I want to ensure that I'm only changing the fields that I want to.  I broke it out to its own function so that I didn't resent everything to the default.

You can see how it's used here: https://github.com/elastic/kibana/pull/100135/files#diff-6a6ff8e750469752a4993b278b7023af4bef43a49ccee59c3933e3d7c68c78f5R26

Also here in a test: https://github.com/elastic/kibana/pull/100135/files#diff-861c70c43696ed0ea535a55a47a316c434a14510046d2eb623d6b228d32e4c8eR211

Reply:
gotcha!